com.teeworlds.Teeworlds.appdata.xml

```diff
@@ -17,6 +18,10 @@ SentUpstream: 2014-09-25
     </p>
   </description>
   <url type="homepage">http://www.teeworlds.com/</url>
+  <categories>
```

Comment:
These should be in the `.desktop` file, no need to duplicate here.

Reply:
I've updated them to match the desktop file. The categories in desktop files and appstream have different usecases so it's valid to include them.